docs/guides/virtualization/cloud-init/01_fundamentals.md

```diff
@@ -0,0 +1,89 @@
+---
+Title: I. cloud-init Fundamentals: The Architecture of First Boot
+author: Wale Soyinka
+contributors:
+tags:
+  - cloud-init
+  - cloud
+  - automation
+---
+
+
+# **I. cloud-init Fundamentals: The Architecture of First Boot**
+
```

Comment:
Hello @wsoyinka Nice job on the content! The level 1 heading here is not needed. It will come down from the meta "Title:" and populate. This line should just be deleted.  Second, in all of these headings, I don't think the additional bold facing works. Each heading level has its own emphasis, so adding bold to the headings is not needed. That is throughout all of these documents.